server/ai_http.go

```diff
@@ -229,16 +228,18 @@ func (h *lphttp) StartLiveVideoToVideo() http.Handler {
 		publishUrlOverwrite := overwriteHost(h.node.LiveAITrickleHostForRunner, subUrl)
 
 		workerReq := worker.LiveVideoToVideoParams{
-			ModelId:      req.ModelId,
-			PublishUrl:   publishUrlOverwrite,
-			SubscribeUrl: subscribeUrlOverwrite,
-			EventsUrl:    &eventsUrlOverwrite,
-			ControlUrl:   &controlUrlOverwrite,
-			Params:       req.Params,
+			ModelId:          req.ModelId,
+			PublishUrl:       publishUrlOverwrite,
+			SubscribeUrl:     subscribeUrlOverwrite,
+			EventsUrl:        &eventsUrlOverwrite,
+			ControlUrl:       &controlUrlOverwrite,
+			Params:           req.Params,
+			GatewayRequestId: &gatewayRequestID,
+			StreamId:         &streamID,
```

Comment:
Should we also add the orchestrator generated requestID as well?

Reply:
👍 I'll do that in a further PR once I get this tidied up